src/App/Model.elm

```diff
@@ -79,20 +79,22 @@ type alias Flags =
     }
 
 
-type alias Model =
-    { currentPage : Page
-    , flags : Flags
-    , pages :
-        { home :
-            { promotedArticles : WebData (List Article)
-            , promotedRecipes : WebData (List Recipe)
-            , recipes : WebData (List Recipe)
-            }
-        , articles : WebData (List Article)
-        , recipes : Dict String (WebData (List Recipe))
-        , recipe : { recipe : WebData Recipe, recipes : WebData (List Recipe) }
-        , contact : ContactForm
+type PageModel
```

Comment:
I'm not a fan of this -- it's putting a lot of info into a single type. The `Page` could (and I think should) be a very [simple](https://github.com/Gizra/drupal-elm-starter/blob/f93745bfcab3e3d28a38ff20838d97482aed1eea/client/src/elm/App/PageType.elm#L11) unio type just holding the notion of what page we are in.

Later, in your `App.Model.model` you will hold a property per [page](https://github.com/Gizra/drupal-elm-starter/blob/f93745bfcab3e3d28a38ff20838d97482aed1eea/client/src/elm/App/Model.elm#L62-L63)

I encourage you to have a look at the way we structured the above linked repo, as it's the same way we structure much larger SPAs.